forge-gui/res/cardsfolder/upcoming/smugglers_share.txt

```diff
@@ -0,0 +1,11 @@
+Name:Smuggler's Share
+ManaCost:2 W
+Types:Enchantment
+T:Mode$ Phase | Phase$ End of Turn | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ At the beginning of each end step, draw a card for each opponent who drew two or more cards this turn, then create a Treasure token for each opponent who had two or more lands enter the battlefield under their control this turn.
+SVar:TrigDraw:DB$ Draw | NumCards$ X | SubAbility$ DBTreasure
+SVar:DBTreasure:DB$ Token | TokenAmount$ Y | TokenScript$ c_a_treasure_sac
+SVar:X:PlayerCountOpponents$ConditionGE2 CardsDrawn
+SVar:Y:PlayerCountOpponents$ConditionGE2 Valid Land.YouCtrl+ThisTurnEntered
```

Comment:
@Northmoc This one is problematic because of:
if they used a fetch land, then two lands entered the battlefield but only 1 land is there for valid